rcl/src/rcl/client.c

```diff
@@ -211,9 +211,7 @@ rcl_client_get_service_name(const rcl_client_t * client)
   return client->impl->rmw_handle->service_name;
 }
 
-/* *INDENT-OFF* */
-#define _client_get_options(client) &client->impl->options;
-/* *INDENT-ON* */
+#define _client_get_options(client) & client->impl->options;
```

Comment:
This was on purpose. I don't think the new version handles this correctly still.

Reply:
In each of the cases below as well where a reference is used in the macro.

Reply:
> I don't think the new version handles this correctly still.

Can you clarify what you mean with this? The current patch passes fine with the current `uncrustify` version. That is why I removed the markers around the line.

Reply:
It's not very important, but there shouldn't be a space between the `&` and the code, that's why the `INDENT-OFF` was there before.

Reply:
If you prefer that I can revert that change.

Reply:
You can or not, like I said not that important.